src/OVAL/probes/fsdev.c

```diff
@@ -97,37 +94,36 @@ static int is_local_fs(struct mntent *ment)
 		return 0;
 	}
 
-	if (ment->mnt_fsname == NULL) {
-		return 0;
+	const char *fstype = ment->mnt_type;
+	if (oscap_str_startswith(fstype, "fuse.")) {
+		fstype += strlen("fuse.");
+	}
+	const char *network_fs[] = {
+		"afs",
+		"ceph",
+		"cifs",
+		"smb3",
+		"smbfs",
+		"sshfs",
+		"ncpfs",
+		"ncp",
+		"nfs",
+		"nfs4",
```

Comment:
We have an issue with `gpfs` filesystems, so I would add that one here. This link indicates that `gpfs` is the right type string: https://www.ibm.com/developerworks/systems/library/es-gpfs/index.html

Reply:
@matejak I have added gpfs to the list.